_posts/0000-01-02-glerby.md

```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![angry-cat](https://cloud.githubusercontent.com/assets/16547949/25400569/064d7bc6-29c1-11e7-90dc-fef44edd9f0a.jpg)
 {: .center}
 
-Caption here
+Your Tears Give Us Strength
```

Comment:
@glerby This is a beautiful line, but I don't like to cry, so I cannot give you much strength

Reply:
More comments